modules/docs/mdx/7.0-MIGRATION-GUIDE.mdx

```diff
@@ -167,3 +169,25 @@ main `react` package.
 - `Box` was promoted from `@workday/canvas-kit-labs/common` to `@workday/canvas-kit-react/layout`
 - `Flex` was promoted from `@workday/canvas-kit-labs/layout` to `@workday/canvas-kit-react/layout`
 - `Stack` was promoted from `@workday/canvas-kit-labs/layout` to `@workday/canvas-kit-react/layout`
+
+## Tokens Updates
+
+### Depth Tokens
+
+**Changes:**
+
+- Depth `inset` was removed from token list.
+- Added new depth: `none` (to remove default depth from component), `5` and `6`.
+- Now all depth will add only `box-shadow` with 2 shadows style and no border.
+
+Also we changed default depth for the next components:
+
+- Card: Depth 2 → Depth 1
+- Select Menu, Menu: Depth 2 → Depth 3
+- Breadcrumbs Dropdown Menu: Depth 2 → Depth 3
+- Combobox: Depth 1 → Depth 3
+- Popup: Depth 2 → Depth 5
+- Dialog: Depth 2 → Depth 5
+- Color Picker Palette, Toast: Depth 2 → Depth 5
```

Comment:
Q: Should "Color Picker Palette" and "Toast" be on separate lines?

Reply:
hmm yeah I would just separate rather than group them for a little more clarity!

Reply:
Thanks for catching it. It should be on separate lines